README.md

```diff
@@ -75,30 +84,15 @@ julia> elements[1:4]
  Element(Beryllium)
  ```
 
-### View the Periodic Table!
-```julia
-julia> elements
-Elements(…119 elements…):
-H                                                  He
-Li Be                               B  C  N  O  F  Ne
-Na Mg                               Al Si P  S  Cl Ar
-K  Ca Sc Ti V  Cr Mn Fe Co Ni Cu Zn Ga Ge As Se Br Kr
-Rb Sr Y  Zr Nb Mo Tc Ru Rh Pd Ag Cd In Sn Sb Te I  Xe
-Cs Ba    Hf Ta W  Re Os Ir Pt Au Hg Tl Pb Bi Po At Rn
-Fr Ra    Rf Db Sg Bh Hs Mt Ds Rg Cn Nh Fl Mc Lv Ts Og
-Uue                                                   
-      La Ce Pr Nd Pm Sm Eu Gd Tb Dy Ho Er Tm Yb Lu    
-      Ac Th Pa U  Np Pu Am Cm Bk Cf Es Fm Md No Lr
-```
-
-## Data by
-* [Periodic-Table-JSON](https://github.com/Bowserinator/Periodic-Table-JSON)
+### Data by
+The data used for this package has been pulled up in parts from [here](https://github.com/Bowserinator/Periodic-Table-JSON)
+Some information has been (and will be) added over the period of time.
```

Comment:
"Some information has been (and will be) added over time." sounds better to me.

Also there seems to be a dot missing after the `here` link.